Tools/dependencies.sh

```diff
@@ -3,21 +3,17 @@ set -xe
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 cd $DIR/..
 
-which brew || /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
```

Comment:
This is for stand-alone developer workstations as well as for the autobuild. If brew is in the path, it won't install. This has no speed penalty. Please leave it there.